apps/nbs-gateway/src/test/java/gov/cdc/nbs/security/oidc/NBS6LoginRouteLocatorConfigurationTest.java

```diff
@@ -0,0 +1,91 @@
+package gov.cdc.nbs.security.oidc;
+
+import com.github.tomakehurst.wiremock.junit5.WireMockExtension;
+import gov.cdc.nbs.gateway.GatewayApplication;
+import gov.cdc.nbs.gateway.home.HomeService;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.ApplicationContext;
+import org.springframework.security.test.web.reactive.server.SecurityMockServerConfigurers;
+import org.springframework.test.web.reactive.server.WebTestClient;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.ok;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathEqualTo;
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig;
+import static org.springframework.security.test.web.reactive.server.SecurityMockServerConfigurers.springSecurity;
+
+@SpringBootTest(
+    classes = {GatewayApplication.class, NBS6LoginRouteLocatorConfiguration.class},
+    webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
+    properties = {
+        "nbs.gateway.classic=http://localhost:10000"
+    }
+)
+class NBS6LoginRouteLocatorConfigurationTest {
```

Comment:
nice!

Reply:
This is as far as i'll be able to go without setting up a Keycloak containerin the test